desktop/cmp/panel/impl/PanelHeader.js

```diff
@@ -45,8 +45,13 @@ export const panelHeader = hoistCmp.factory({
                             item: title
                         }) :
                         filler(),
-                    ...(!collapsed ? headerItems : []),
-                    collapseButton({panelModel})
+                    hbox({
```

Comment:
Why is the double click on the grid column chooser even getting here?  
Does that mean we are doing something wrong in our handling of the  column chooser or the popover? 

Seems to me that:

-If the column chooser is using the double click, it should certainly consume it. 
-The popover should not be sending it

Reply:
I think the issue is that all items in the panel header are valid targets for the panel header's `onDoubleClick` event handler, including items currently passed in through the `headerItems` prop. So no matter what item lies there in headerItems, whether it's something static like text or dynamic like a button or a column chooser, we don't want `onDoubleClick` to propagate to the parent. So we are now wrapping the headerItems in its own div and stopping any double click events from toggling the panel.